xla/service/gpu/gpu_windowed_einsum_handler.cc

```diff
@@ -697,7 +696,7 @@ class WindowedEinsumVisitor : public DfsHloRewriteVisitor {
     // to be consumed by the dot. This is advantageous since the chained dot can
     // fully utilize all the resources on the GPU while comm is hidden by the
     // first collective matmul loop.
-    for (GpuWindowedEinsumHandler::WindowedEinsumAgLoops ag_loop :
+    for (GpuWindowedEinsumHandler::WindowedEinsumAgLoops& ag_loop :
```

Comment:
Can we factor out changes like this into a separate PR? Itr's unrelated and makes the diff hard to parse.